packages/deployments/contracts/contracts/core/connext/helpers/OZERC20.sol

```diff
@@ -248,8 +250,8 @@ contract ERC20 is IERC20, IERC20Permit, EIP712 {
 
     _beforeTokenTransfer(address(0), _account, _amount);
 
-    supply = supply.add(_amount);
-    balances[_account] = balances[_account].add(_amount);
+    supply = supply + _amount;
+    balances[_account] = balances[_account] + _amount;
     emit Transfer(address(0), _account, _amount);
```

Comment:
should  be:

```ts
        _totalSupply += amount;
        unchecked {
            // Overflow not possible: balance + amount is at most totalSupply + amount, which is checked above.
            _balances[account] += amount;
        }
        emit Transfer(address(0), account, amount);

        _afterTokenTransfer(address(0), account, amount);
```

[source](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/e0137d4cbe14ba6be2b52c6ecfb006b515c6f780/contracts/token/ERC20/ERC20.sol#L259-L272)